ci-operator/config/openshift-knative/eventing-istio/openshift-knative-eventing-istio-release-v1.15__415.yaml

```diff
@@ -1,8 +1,24 @@
+base_images:
+  ocp_ubi-minimal_8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  openshift_release_rhel-8-release-golang-1.22-openshift-4.17:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.22-openshift-4.17
 build_root:
   project_image:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
 images:
 - dockerfile_path: openshift/ci-operator/knative-images/controller/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - $GO_RUNTIME
```

Comment:
Do we set those somewhere else?

Reply:
Through hack's image template: https://github.com/openshift-knative/hack/blob/main/cmd/generate/dockerfile-templates/DefaultDockerfile.template

Reply:
Yes, now every component defines those or is about to start defining them